features/cellular/framework/targets/QUECTEL/BC95/QUECTEL_BC95_CellularNetwork.cpp

```diff
@@ -38,3 +38,26 @@ nsapi_error_t QUECTEL_BC95_CellularNetwork::set_access_technology_impl(RadioAcce
 
     return NSAPI_ERROR_OK;
 }
+
+nsapi_error_t QUECTEL_BC95_CellularNetwork::clear()
```

Comment:
Camelcase function name ? Strictly we shouldn't have those but it seems there are similar ones already so consistency is more important..